docs/FAQ.md

```diff
@@ -1,15 +1,159 @@
+manner.  To make a component "trusted", you must establish some root of trust.
+A Root of Trust Measurement is the process of measuring (hashing) components on
+a system to validate their integrity. When establishing a root of trust, there
+are two primary schools of thought: Static Root of Trust Measurements and
+Dynamic Root of Trust Measurements.
+
+### SRTM
+
+SRTM takes place at system boot from firmware / bios. This process requires
+measurements of the entire boot sequence at once. This is both a good and a
+bad thing, because any changes to any part of the process (including even
+bios settings) would cause the entire system to need to be remeasured. SRTM
+Starts with Core Root of Trust for Measurement (CRTM), and works its way up the chain where
+each link verifies the next. In other words, SRTM starts by measuring the bios
+boot block and then moves on to measure the rest of the bios.
+
+### DRTM
+
+DRTM aims to address shortcomings of SRTM. DRTM creates trusted environment from
+an untrusted state. Essentially what this means is that regardless of weather
+or not computer was booted from trusted software, it uses hardware to ensure
+that no malicious software has been run. For this reason, DRTM takes place
+after boot, as opposed to SRTM, which takes place pre-boot. Intel's solution
+for DRTM is TXT, which utilizes special instructions on the processor (SMX).
+AMD's solution for DRTM is called SKINIT. Trenchboot provides a solution for
+setting up DRTM for an arbitrary platform.
+
+### Typical Measured Launch Process
+
+The process for carrying out DTRM depends on your system:
+
+- [Intel TXT: Software Development Guide][txt-spec]
+- [System Programming Volume of for AMD64 (PDF)][skinit-spec] (see
+  "Secure Startup with SKINIT" section)
+
+[txt-spec]: https://www.intel.com/content/www/us/en/content-details/315168/intel-trusted-execution-technology-intel-txt-software-development-guide.html
+[skinit-spec]: https://www.amd.com/system/files/TechDocs/24593.pdf
+
+### So, why TrenchBoot?
+
+- Managing multiple independent components individually can be confusing
+- Most DRTM systems are platform-dependent
+- Most DRTM systems can only measure entities already present in RAM at the
+    time DRTM sequence is initiated
+- Most DRTM systems support only one attestation action
+
+## 2. How does TrenchBoot work? (trenchboot architecture)
+
+### Architecture Overview
+
+*The full architecture overview can be found on
+[the corresponding page](./theory/Architecture.md).*
+![Overview](img/Architectural_Flow.png)
+This diagram shows a general overview of the overall architecture of a
+TrenchBoot system. In Bootstrap phase the system environment is measured and
+that information stored in the dynamic PCRs, using whatever method is
+appropriate for the given hardware. The intermediate phase analyzes the
+information collected in the bootstrap phase to decide if it wants to continue
```

Comment:
Change "if it wants to" to "whether" and remove comma after OS on next line.

Edited-by: Amanda Ferguson [af.jenks@gmail.com](mailto:af.jenks@gmail.com)